CITATION.cff

```diff
@@ -14,6 +14,6 @@ date-released: "2022-11-16"
 identifiers:
   - description: This is the archived snapshot of version 1.0.0 of IXI025
     type: doi
-    value: "10.5281/zenodo.6563519"
+    value: "10.5281/zenodo.7331083"
```

Comment:
so this reference is https://zenodo.org/record/7331083

Reply:
yes